### ISSUE TITLE
Action execution failed Error calling startBuild: Cannot have more than 0 builds using brand new account/org

### ISSUE THREAD

**Issue by kbroughton** [aws-samples/aws-secure-environment-accelerator#982]
**Required Basic Info**

* Accelerator Version: v1.5.1-a
* clean run
* Which State did the Main State Machine Fail in: (e.g. N/A, Phase 0)
  Failed at the Pipeline stage before State Machine is ready
  
* config.json based on reference-artifacts/SAMPLE_CONFIGS/config.lite-CTNFW-example.json
see config-redacted.json.log attached

**Describe the bug**
ASEA-InstallerPipeline fails stating account limit exceeded - cannot have more than 0 builds in queue

**Failure Info**

```
Action execution failed
Error calling startBuild: Cannot have more than 0 builds in queue for the account (Service: AWSCodeBuild; Status Code: 400; Error Code: AccountLimitExceededException; Request ID: f0d98e36-8ac5-4668-abab-2108afa3251e; Proxy: null)
Required files
```

If a CodeBuild step failed- please provide the full CodeBuild Log
* Could not find

**Steps To Reproduce**

* Create a brand new account. Make it an Org. Follow all the steps in https://catalog.us-east-1.prod.workshops.aws/workshops/f3ed5d0f-d2f1-47e8-a305-168da9179aaa/en-US/sea-administrators
to setup the org and root account.
* Upload the attached file to an S3 bucket
* Launch the AWSA CF template using latest release

**Expected behavior**
Expected it runs to completion

**Screenshots**
![Screen Shot 2022-05-25 at 3 21 31 PM](https://user-images.githubusercontent.com/10562482/170403237-ab98647b-7a88-4f83-afc3-7c86355ad096.png)
[config-redacted.json.log](https://github.com/aws-samples/aws-secure-environment-accelerator/files/8775781/config-redacted.json.log)

**Comment by Brian969** [comment]
Hi,
- On quick review of the config file, it is not valid json (run it through any json validator or parser, [one](https://jsonformatter.curiousconcept.com/#) example) and needs to be fixed as a first step (you can't break lines the way you did).
- I also noticed you started with the Control Tower + NFW based config file, yet disabled the primary Control Tower compatibility settings.  Are you installing on top on Control Tower or standalone?  
  - If doing a standalone install, I do not think you will get the desired outcome and you should use config.lite-NFW-example.json as your starting point (identical but without all the Control Tower settings).
  - if deploying with control Tower you cannot set "ct-baseline": **false**, and "control-tower-supported-regions" must have at least one region
- Finally, while the above are concerns, likely not the cause of this initial failure - "**account limit exceeded**"
  -I'm guessing you have a very new AWS account, as it appears to have unusually low initial limits established
  - Can you start by running an small EC2 instance in the account (i.e. t2.micro) in your home or installation region for 15minutes.  This should force the account to establish some basic limits.
  - Before you rerun the ASEA, you will need to delete the EC2 instance and all associated sub-objects like security groups or the state machine will purposely fail attempting to delete the default VPC.  To ensure you properly cleaned up, recommend you manually delete the default VPC in that region.
   - if you still have trouble with errors associated to limits - you'll likely need to reach out to your local AWS account team to help you get your limits increased.

Hope this helps!

**Comment by kbroughton** [comment]
Thanks for the speedy reply @Brian969 !

The config problem was due to copy/paste from github - the file was too large and caused format problems.
I switched to NFW and tried to launch a t2.mediium.
I skipped over that part in the docs cause i assumed it was just for lab purposes. It would be great to add an explainer that it's to "activate" the account and you will hit limit issues on a new account for 2-3 days.

It actually took 60 hrs from my first activity in the account to get the t2 to launch. I opened 3 support tickets and the SLA said 48 hours for bumping limits, so I might have been unlucky.

Anyway, once that was cleared up, the pipeline ran to completion and I'm now debugging state-machine errors. 
```
  "cause": {
    "errorType": "Error",
    "errorMessage": "This execition requires Accelerator execution with scope: \"FULL\"",
    "trace": [
      "Error: This execition requires Accelerator execution with scope: \"FULL\"",
      "    at I2e (/var/task/index.js:1392:22966)",
      "    at processTicksAndRejections (internal/process/task_queues.js:95:5)",
      "    at async Runtime.dT0 [as handler] (/var/task/index.js:1392:17569)"
    ]
  }
  ```

I'll close this out and open a new one on that topic

thanks again
